src/locales/de.json

```diff
@@ -0,0 +1,192 @@
+{
+  "snippets": {
+    "info": "Info",
+    "survPerkConfig": "Überlebenden-Perks konfigurieren",
+    "killPerkConfig": "Killer-Perks konfigurieren",
+    "globalSwitchText": "Alle an/aus",
+    "startPerkRoll": "Klicke zum Starten auf einen Perkslot",
+    "buttonHome": "Startseite",
+    "buttonKillRoulette": "Killer-Roulette",
+    "buttonSurvRoulette": "Überlebenden-Roulette"
+  },
+  "about": {
+    "header": "Worum geht es hier?",
+    "description": "Diese Seite ist ein Perk-Roulette für das Spiel {url}. Um genau zu sein ist es kein Roulette, sondern eine Slot-Maschine, wie ein Glücksspieleautomat. In diesem Video erfährst Du alles, was Du wissen musst:",
+    "videoUrl": "https://www.youtube-nocookie.com/embed/vSaUjJC68Kc"
+  },
+  "howItWorks": {
+    "header": "Wie funktioniert das?",
+    "description": "Falls Du alle Perks die es gibt für einen Überlebenden oder Killer hast, klicke einfach auf einen der roten Buttons oben, klicke auf einen der Perk-Slots, fertig. Daraufhin werden vier zufällige Perks für Dich ausgewählt, mit denen Du dann spielst. Genau so funktioniert ein Perk-Roulette. Viel Spaß!",
+    "missingPerks": {
+      "header": "Aber ich habe nicht alle Perks!!1elf",
+      "description": "Falls Dir Perks fehlen, klicke auf eine der \"... konfigurieren\"-Boxen unten. Dann klicke auf die Perks die Du nicht hast oder nicht willst. Dine Auswahl wird nicht auf dem Sevrer gespeichert, sondern eine individuelle URL zusammengebaut, die Du Dir nach dem Konfigurieren in Deinen Lesezeichen speichern kannst. Also Perks konfigurieren, als Lesezeichen speichern, fertig."
```

Comment:
- D**e**ine Auswahl wird nicht auf dem Sevrer gespeichert: Es wird eine individuelle URL zusammengebaut, die Du Dir nach dem Konfigurieren in Deinen Lesezeichen speichern kannst.